crates/futures/tests/tests.rs

```diff
@@ -155,3 +155,21 @@ async fn can_use_an_async_iterable_as_stream() {
     assert_eq!(stream.next().await, Some(Ok(JsValue::from(24))));
     assert_eq!(stream.next().await, None);
 }
+
+#[wasm_bindgen_test]
+#[should_panic]
+async fn should_panic() {
+    panic!()
+}
+
+#[wasm_bindgen_test]
+#[should_panic = "error message"]
+async fn should_panic_string() {
+    panic!("error message")
+}
+
+#[wasm_bindgen_test]
+#[should_panic(expected = "error message")]
+async fn should_panic_expected() {
+    panic!("error message")
+}
```

Comment:
Why are these tests a part of `wasm-bindgen-futures`'s test suite?

Reply:
I was testing it with async function to make sure that the parsing is correct.